### ISSUE TITLE
GWT 2.8.1 unusable-by-js WARNING

### ISSUE THREAD

**Issue by HoochSDS** [GwtMaterialDesign/gwt-material#673]
Hi, I switched from GWT 2.8.0 to GWT 2.8.1
Now I'm facing following warnings.
The Application is running without errors, so far.
Using: GMD 2.0-SNAPSHOT
Thanks
```
[INFO] --- gwt-maven-plugin:2.8.1:compile (default) @ app---
[INFO] Compiling module sds.solution.App
[INFO]    Warnings in gwt/material/design/client/js/JsMaterialElement.java
[INFO]       [WARN] Line 170: [unusable-by-js] Type of parameter 'value' in 'JsMaterialElement JsMaterialElement.set(String, Date)' is not usable by but exposed to JavaScript.
[INFO]       [WARN] Line 173: [unusable-by-js] Type of parameter 'value' in 'JsMaterialElement JsMaterialElement.set(String, Date, Functions.Func)' is not usable by but exposed to JavaScript.
[INFO]    [WARN] Suppress "[unusable-by-js]" warnings by adding a `@SuppressWarnings("unusable-by-js")` annotation to the corresponding member.

```

**Comment by kevzlou7979** [comment]
Ok this will be fixed on upcomming 2.0-rc7 release

**Comment by kevzlou7979** [comment]
Fixed via https://github.com/GwtMaterialDesign/gwt-material/commit/6343e830951bf811fc78cf276c7491f71e8c80c9
